docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,337 @@
+Stable Patch Format
+===================
+
+Destination
+-----------
+
+Stable patches must be sent to: kernel-team@lists.ubuntu.com
+
+.. important::
+
+   please subscribe `here <https://lists.ubuntu.com/mailman/listinfo/kernel-team>`__
```

Comment:
If this is a requirements, consider making it a Prerequisite instead of an alert box.